packages/expo-module-scripts/package.json

```diff
@@ -34,7 +34,7 @@
     "babel-preset-expo": "~8.2.3",
     "commander": "^2.19.0",
     "eslint-config-universe": "^3.0.2",
-    "find-yarn-workspace-root": "^1.2.1",
+    "find-yarn-workspace-root": "~2.0.0",
```

Comment:
We use the semver caret `^` except in a few places where the minor version sometimes can have a breaking change (our Expo modules and ts-jest). Is there a deliberate reason to use the tilde here?

Reply:
Don't want to accidentally mismatch these versions in the future. They caused a lot of code bloat.